experimental/Schemes/BlowupMorphism.jl

```diff
@@ -96,108 +97,223 @@ function strict_transform(p::BlowupMorphism, inc::CoveredClosedEmbedding)
   return Z_trans, inc_Z_trans, pr_res
 end
 
+## THIS SHOULD GO TO mpolyquo-localization.jl
+## it is the localized and/or quotient wrapper to the MPolyIdeal-method
+function saturation_with_index(I::T,J::T) where T <: Union{ MPolyQuoIdeal, MPolyLocalizedIdeal, MPolyQuoLocalizedIdeal}
+  R = base_ring(I)
+  R == base_ring(J) || error("Ideals do no live in the same ring.")
+
+  I_sat = saturated_ideal(I)
+  J_sat = saturated_ideal(J)
+
+  I_result,k = Oscar.saturation_with_index(I_sat,J_sat)  ## why is saturation_with_index not exported from Rings/mpoly-ideal.jl?
```

Comment:
```suggestion
  I_result,k = saturation_with_index(I_sat,J_sat)  ## why is saturation_with_index not exported from Rings/mpoly-ideal.jl?
```

Reply:
Feel free to export it if it makes sence. Please add it to the documentation as well.

Reply:
I put the exports in mpoly-ideals.jl, but I feel that they definitely belong somewhere else, because there are no further export statements in that file. Can you advise?